feature-libs/customer-ticketing/core/customer-ticketing-constants.ts

```diff
@@ -1 +1,2 @@
+export const MAX_INPUT_CHARACTERS_FOR_MESSAGE = 5000;
 export const MAX_INPUT_CHARACTERS = 2000;
```

Comment:
Limit is 5000 on each place in mockups, I didn't check and put 2000 initially.
You can just modify `MAX_INPUT_CHARACTERS` to 5000.

Unless, we think we should provide different limit for input message and reopen/close message.
IMO, it should be same as we are treating it as just another message to create/close/reopen ticket.

Reply:
Same thing for the input characters as well

Reply:
So, we need to remove one of the constants

Reply:
since these are just constants and will be re-used in root, let us move this in root. A few more variable won't save us more bytes

Reply:
related to https://github.com/SAP/spartacus/pull/16063#discussion_r955323891